.github/workflows/ci.yml

```diff
@@ -33,26 +33,28 @@ jobs:
 
       - name: Run tests
         run: |
-          poetry run coverage run -m pytest
-          poetry run coveralls
+          poetry run tox
         env:
           OPENAI_API_KEY: "sk-fake-test-key-123"
 
       - name: Coveralls Parallel
-        uses: coverallsapp/github-action@v2
-        with:
-          github-token: ${{ secrets.GITHUB_TOKEN }}
-          parallel: true
-          flag-name: django-coverage
-          allow-empty: true
+        run: |
+          poetry run coveralls
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          COVERALLS_SERVICE_NAME: "github-actions"
+          COVERALLS_REPO_TOKEN: ${{ secrets.COVERALLS_REPO_TOKEN }}
+          COVERALLS_FLAG_NAME: django-coverage-${{ matrix.python-version }}
```

Comment:
I guess you could run coverage only on the last version, but since this is working, it's fine